src/modules.js

```diff
@@ -232,8 +232,8 @@ function infermedicaModule() {
 
   const initializeBrowser = async () => {
     if (browser === null) {
```

Comment:
Q: is it `null` by default? 🤔

Reply:
[Line 202](https://github.com/infermedica/modular-analytics/blob/4b36b65064522c2b7a61e1d406ec27e35af0e9b9/src/modules.js#L202)